apps/bfd-pipeline/bfd-pipeline-shared-utils/src/main/java/gov/cms/bfd/pipeline/sharedutils/DatabaseOptions.java

```diff
@@ -23,53 +21,23 @@ public DatabaseOptions(String databaseUrl, String databaseUsername, char[] datab
     this.databaseUrl = databaseUrl;
     this.databaseUsername = databaseUsername;
     this.databasePassword = databasePassword;
-    this.databaseDataSource = null;
   }
 
-  /**
-   * Constructs a new {@link DatabaseOptions} instance.
-   *
-   * @param databaseDataSource the value to use for {@link #getDatabaseDataSource()}
-   */
-  public DatabaseOptions(DataSource databaseDataSource) {
-    this.databaseUrl = null;
-    this.databaseUsername = null;
-    this.databasePassword = null;
-    this.databaseDataSource = databaseDataSource;
-  }
-
-  /**
-   * @return the JDBC URL of the database to load into, or <code>null</code> if {@link
-   *     #getDatabaseDataSource()} is used, instead
-   */
+  /** @return the JDBC URL of the database to load into */
   public String getDatabaseUrl() {
     return databaseUrl;
   }
 
-  /**
-   * @return the database username to connect as when loading data, or <code>null</code> if {@link
-   *     #getDatabaseDataSource()} is used, instead
-   */
+  /** @return the database username to connect as when loading data */
   public String getDatabaseUsername() {
     return databaseUsername;
   }
 
-  /**
-   * @return the database password to connect with when loading data, or <code>null</code> if {@link
-   *     #getDatabaseDataSource()} is used, instead
-   */
+  /** @return the database password to connect with when loading data */
```

Comment:
Arrays are always mutable so this isn't really safe.  Also the array winds up being turned back into a String anyway when it's used so this is just adding extra complexity.

Reply:
Thanks! Done.